src/pages/DataSubmissionForm.js

```diff
@@ -99,6 +103,17 @@ export const DataSubmissionForm = () => {
     formFiles[key] = value;
   };
 
+  const onValidationChange = ({ key, validation }) => {
+    /* eslint-disable no-console */
+    console.log('Validation OnChange:', key, validation);
```

Comment:
nit: remove before merging

Reply:
👍 I added it to match with the logging in `onChange`, but I can remove them all, to be honest. I think it's far enough in the development process to not need debugging.